libc/utils/gpu/server/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ target_include_directories(llvmlibc_rpc_server PUBLIC ${CMAKE_CURRENT_SOURCE_DIR
 # Ignore unsupported clang attributes if we're using GCC.
 target_compile_options(llvmlibc_rpc_server PUBLIC
                        $<$<CXX_COMPILER_ID:GNU>:-Wno-attributes>)
+target_compile_definitions(llvmlibc_rpc_server PUBLIC
```

Comment:
@jhuber6 It seems to me that you don't need this if you move [libc/CMakeLists.txt](https://github.com/jhuber6/llvm-project/blob/d2f4ac04afa5d464d15af8ed7c3469f3f096204a/libc/CMakeLists.txt#L57-L62) with the block you moved.

Reply:
This part it build with the rest of the LLVM project, so I'd like to avoid global definitions if possible.